backend/WebAPI/Controllers/AdminController.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Application.DTOs.Admin;
+using Application.Interfaces.Services;
+using FluentValidation;
+using FluentValidation.Results;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.ComponentModel.DataAnnotations;
+using WebAPI.Validation.Admin;
+
+namespace WebAPI.Controllers
```

Comment:
Using UserManager > Repository > Service, I think we are doing the same a lot of time, we could even use UserManager in the Controller and it will not be verbose, we should discuss this, if we should use at least Repository or use UserManager directly for all UserIdentity entities like Admin, Client, Company.